migrate/migrations/20200413113132-add-no-notification-alert-log.sql

```diff
@@ -1,6 +1,7 @@
 -- +migrate Up notransaction
 
 ALTER TYPE enum_alert_log_event ADD VALUE IF NOT EXISTS 'no_notification_sent';
+ALTER TYPE enum_alert_log_subject_type ADD VALUE IF NOT EXISTS 'no_notification';
```

Comment:
`no_notitifcation` is the event type, the subject type should be `user` rather than adding a new one.